dev/ubuntu20.04.Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:20.04
+WORKDIR /home/
+
+RUN ["apt-get", "-y", "upgrade"]
+RUN ["apt-get", "-y", "update"]
+RUN ["apt-get", "install", "-y", "git"]
```

Comment:
Each `RUN` statement creates a layer in the Docker image. This can artificially increase the size of images. Official Docker best practice can be found here: https://docs.docker.com/build/building/best-practices/#run

Reply:
After re-reading the best practice, I see there are also caching issues that make chaining `apt-get` `RUN` commands necessary. Wow.

Reply:
Thanks for flagging this, I've seen the pattern of chaining apt-get commands before but didn't realize why.

For what it's worth, I'm not going to do a full apt-get clean here because we do install additional packages in the setup script; if maximally optimizing for size of the image is important, we could install all those packages here and then delete the rest of the apt-get cache, but it seems like [it doesn't save a ton of space](https://gist.github.com/marvell/7c812736565928e602c4?permalink_comment_id=3463716#gistcomment-3463716) so I'm going to save that for later.